pkg/oc/admin/network/isolate_projects.go

```diff
@@ -72,8 +73,12 @@ func (i *IsolateOptions) Run() error {
 
 	errList := []error{}
 	for _, project := range projects {
+		if project.Name == kapi.NamespaceDefault {
```

Comment:
Why does this require a CLI change?  The server should decide what is forbidden as the CLI can always be bypassed.

Reply:
'default' namespace is special because openshift deploys kube service in 'default' namespace by default and by allowing the namespace to be isolated, other namespaces can not access the kube api.